os/board/common/crashdump.c

```diff
@@ -129,6 +133,81 @@ static int get_userfs_address_size(uint32_t *userfs_start, size_t *userfs_size)
 	return 0;
 }
 
+static int get_ext_userfs_part(size_t *size, int *index)
```

Comment:
please provide the description for all of your function.
it will be easier to understand the code for a new person.

Reply:
Since this file doesn't have any other function descriptions either,
I could take this up separately if required.
@sunghan-chang is addition of descriptions required here?